NEWS.md

```diff
@@ -1,3 +1,14 @@
+# azuremlsdk 0.8.0
+## New features
+- Foreach backend for distirbuted training and batch inferencing.
+- R Section in Environment Definition.
+- Added Azure Data Lake Gen2 Datastore support.
+
+## Bug fixes
+- Fix issues with Dataset creation and usage.
+- Fix Interactive Authentication
+
```

Comment:
we should also mention that we've removed the widget function in this version

Reply:
added